src/devices/Mpu9250/samples/Mpu9250.sample.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Threading;
+using Iot.Device.Mpu9250;
+
+namespace DemoMpu9250
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello MPU9250!");
+
+            I2cConnectionSettings mpui2CConnectionSettingmpus = new I2cConnectionSettings(1, Mpu9250.DefaultI2cAddress);
+            Mpu9250 mpu9250 = new Mpu9250(new UnixI2cDevice(mpui2CConnectionSettingmpus));
```

Comment:
Use new I2cDevice.Create instead if package available.

Reply:
that's why we need to make this internal :smile: